api/grpc/mpi/v1/files.proto

```diff
@@ -113,13 +114,13 @@ message FileContents {
 // Meta information about the file, the name (including path) and hash
 message FileMeta {
     // the name of the file
```

Comment:
Could you modify this comment to clearly say that `name` is a path instead of just a file name (even though it's said in the `FileMeta` comment)?

Reply:
sure updated it there now